Today/Controllers/ReminderDetailViewController/ReminderDetailEditDataSource.swift

```diff
@@ -86,17 +87,17 @@ class ReminderDetailEditDataSource: NSObject {
         
         // MARK: - Public Properties
         
-        var sectionTitle: String {
+        public var sectionTitle: String {
```

Comment:
Not using localized version of strings there.

Reply:
Don't put enums randomly in a file. Make a habit of putting at some specific place. We usually place them at the top of the file outside any class if not for any specific Type.

Reply:
Not sure if i got your point here.
I think i am using the localised strings here...
![image](https://user-images.githubusercontent.com/85882612/129354722-bfe61a16-63b6-4f84-8174-ae4bc60b1ff2.png)

Reply:
I usually put nested types at the end of the class/struct.
Right now should i place it in an extension of the class, at the bottom of the file ?

Reply:
These are not localized strings.
Yeah put them outside the class/struct if they are not specific to any class/struct.

Reply:
This enum was required only in a specific class, so i created an extension and defined the enum there.